documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc

```diff
@@ -411,9 +412,19 @@ include::{sourcedir}/basic/BooleanMappingTests.java[tags=basic-boolean-example-i
 ----
 ====
 
-However, it is certainly reasonable to map booleans to other representations in the
-database and Hibernate provides support for mapping booleans to the more well known
-representations - 'Y' / 'N', 'T' / 'F' or 1 (true) / 0 (false).
+But, especially when working with a legacy database, it's quite common to find boolean values
+encoded as a character, or as an integer, even when the platform _does_ support a built-in
+`BOOLEAN` type.
+
+The easiest way to handle this situation is to use a JPA `AttributeConverter`. You can easily
+write your own `AttributeConverter`, but for your convenience, Hibernate comes with three
+built-in converters for common cases:
+
+- `YesNoConverter` encodes a boolean value as `'Y'` or `'N'`,
+- `TrueFalseConverter` encodes a boolean value as `'T'` or `'F'`, and
+- `NumericBooleanConverter` encodes the value as an integer, `1` for true, and `0` for false.
+
+You may specify the converter to use with the `@Convert` annotation.
```

Comment:
Again "you", and here, this would already be covered by the link to the converter section